### PR TITLE
refactor project

### PR DESCRIPTION
**Pull request by HAYATO-Kz**
- Refactor project
- Add Container base component that include header and safeareaview
- Remove native base from project
- **Found some issue about add new font to project (Problem with cocoapod)**
- Change to Thai language

### CODE REVIEW

File: utils/api.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const backendAPI = axios.create({
+  baseURL: 'http://localhost:3000',
```
Comment (NokKbl) [inline utils/api.ts:4]: Good job bro

File: metro.config.js [hunk 2]
```diff
@@ -5,13 +5,41 @@
...
+//   transformer: {
+//     getTransformOptions: async () => ({
+//       transform: {
+//         experimentalImportSupport: false,
+//         inlineRequires: false,
+
+//       },
+//     }),
+//   },
+// };
```
Comment (NokKbl) [inline metro.config.js]: Please remove this if you didn't use it.

File: components/Header/Header.style.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+`;
+
+export const Title = styled.Text`
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 22px;
+  color: #613400;
+`;
+
+export const Wrapper = styled.View``;
```
Comment (NokKbl) [inline components/Header/Header.style.ts]: @HAYATO-Kz Why you need to use styled? It's not styled anything!!

File: components/TouchableIcon/TouchableIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Text } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+
+import { IconWrapper } from './TouchableIcon.style'
+
+interface TouchableIconProps {
+    onPress: () => {}
```
Comment (NokKbl) [inline components/TouchableIcon/TouchableIcon.tsx]: You mean `onPress: () => void` ?

**Comment by NokKbl** [comment]
Reviewed

File: utils/api.ts (hunk 1 above)
Comment (HAYATO-Kz) [reply utils/api.ts:4]: Dafaq?

**Review by NokKbl — APPROVED** [review]

**Pull request merged**